pkg/workload/csv_test.go

```diff
@@ -78,20 +80,25 @@ func TestHandleCSV(t *testing.T) {
 func BenchmarkWriteCSVRows(b *testing.B) {
 	ctx := context.Background()
 
-	var rows [][][]interface{}
+	var batches []coldata.Batch
 	for _, table := range tpcc.FromWarehouses(1).Tables() {
-		rows = append(rows, table.InitialRows.Batch(0))
+		cb := coldata.NewMemBatch(nil)
+		var a bufalloc.ByteAllocator
+		table.InitialRows.FillBatch(0, cb, &a)
+		batches = append(batches, cb)
 	}
 	table := workload.Table{
 		InitialRows: workload.BatchedTuples{
-			Batch: func(rowIdx int) [][]interface{} { return rows[rowIdx] },
+			FillBatch: func(batchIdx int, cb coldata.Batch, _ *bufalloc.ByteAllocator) {
+				*cb.(*coldata.MemBatch) = *batches[batchIdx].(*coldata.MemBatch)
```

Comment:
Is this the reason you had to export MemBatch? This seems kind of unpleasant - could you maybe change the `FillBatch` interface to return a `coldata.Batch` instead of modifying one in place? Or does that have other problems?